google-cloud-datastore/pom.xml

```diff
@@ -178,12 +178,52 @@
       <version>1.4.2</version>
       <scope>test</scope>
     </dependency>
+    <!-- OpenTelemetry -->
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-sdk</artifactId>
       <version>${opentelemetry.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk-common</artifactId>
+      <version>${opentelemetry.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk-trace</artifactId>
+      <version>${opentelemetry.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-semconv</artifactId>
+      <version>1.1.0-alpha</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- END OpenTelemetry -->
+    <!-- Cloud Ops -->
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-trace-v1</artifactId>
+      <version>1.3.0</version>
```

Comment:
please replace this with the `gapic-libraries-bom` (https://github.com/googleapis/google-cloud-java/tree/main/gapic-libraries-bom) which will manage the `trace` versions for us centrally. Also, these versions look pretty old - any reason why? If not, please update to the latest.

Reply:
Done. Updated to use the BOM. No reason for the older version.

Reply:
Wonder why this is still [failing](https://github.com/googleapis/java-datastore/actions/runs/9067200287/job/24912269280)
```
[INFO] --- maven-dependency-plugin:3.5.0:analyze (default-cli) @ google-cloud-datastore ---
Error:  Used undeclared dependencies found:
Error:     com.google.api.grpc:proto-google-cloud-trace-v1:jar:2.43.0:test
```

Reply:
Ok so looks like we still need to list the dependencies w/o the versions.